### PR TITLE
add support for eslint, clean up directory

### PR DESCRIPTION
**Pull request by bentoncase**
I restructured the files so that it will be much easier to use browserify and npm to pull in third party packages in the future. One of my upcoming pull requests will do this.

I also added support for eslint, and it looks like there are about 500 errors in content.js. @groovecoder what should we do about this? Fix linting errors in a separate pull request?

### CODE REVIEW

**Review by groovecoder — CHANGES_REQUESTED** [review]
Hmm ... since this code is copied from OpenWPM and Privacy Badger, I don't know that we want to enforce those cleanups. Probably just skip the `content.js` file unless/until we start changing its contents ourselves.

We also need to update the "Run it" section of the README to be `web-ext run -s src/`

**Comment by bentoncase** [comment]
@groovecoder how does it look now? I removed the readme comment about web-ext since it's now a dev dependency

File: README.md [hunk 1]
```diff
@@ -14,13 +14,10 @@ anything in a database. If you need all that good stuff, you should really look
...
 ## Run it
-
-You should really use the [`web-ext`](https://www.npmjs.com/package/web-ext)
-tool - it's great. After you've installed it globally ...
-
 ```
 cd data-leak
-web-ext run
+npm install
+npm run build
```
Comment (groovecoder) [inline README.md:22]: This builds a `.zip` but doesn't actually run the browser with the add-on. Let's keep it `web-ext run -s src`

**Review by groovecoder — CHANGES_REQUESTED** [review]

**Comment by bentoncase** [comment]
@groovecoder r?

**Review by groovecoder — APPROVED** [review]
Technically the build step isn't necessary, but this works.

**Pull request merged**